### PR TITLE
Build/update workflow to run cucumber tests

### PR DESCRIPTION
**Pull request by mrahanjam**
# Update Github Workflow

## Description
This PR updates the workflow to run the cucumber tests.

## PR Type
- [ ] Bug Fix
- [ ] Feature
- [X] Build Fix
- [ ] Testing
- [ ] General Improvement
- [ ] Cherry Pick

## Links
Jira: N/A

## Test Plan
Manual

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -55,34 +52,23 @@ jobs:
...
         yarn run bower-root # Install Bower dependencies
+
     - name: Start CDAP-UI and run the tests
       run: |
-        Xvfb :1 -screen 0 1024x768x24 & export DISPLAY=:1 # Set Display port
         yarn run cdap-full-build-more-memory # Build UI
         yarn start & > server.out
         yarn run test:unit
-        yarn run test:integration
+        mvn clean verify -P e2e-tests
```
Comment (njbriggs) [inline .github/workflows/build.yml]: Nit: newline at EOF

File: sandboxjs/sandbox_starter.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+def run_shell_command(cmd):
+    process = subprocess.run(cmd.split(" "), stderr=subprocess.PIPE)
+    if process.returncode != 0:
+        print("Process completed with error: ", process.stderr)
+    assert process.returncode == 0
+
+# Start CDAP sandbox
+print("Downloading CDAP sandbox")
+sandbox_url = "https://builds.cask.co/artifact/CDAP-BUT/shared/build-latestSuccessful/SDK/cdap/cdap-standalone/target/cdap-sandbox-6.8.0-SNAPSHOT.zip"
```
Comment (de-lan) [inline sandboxjs/sandbox_starter.py]: So for this sandbox url, we need to update this whenever a new snapshot is released?

Comment (mrahanjam) [reply sandboxjs/sandbox_starter.py]: Yes, at least for now. I'll look for a way to see if we can get it automatically (similar to what we had in `setup_gitpod.js`.

**Review by de-lan — APPROVED** [review]
LGTM

File: pom.xml [hunk 3]
```diff
@@ -1,13 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
   Copyright © 2014-2019 Cask Data, Inc.
```
Comment (sumengwang) [inline pom.xml]: copyright year

**Review by itsankit-google — COMMENTED** [review]
I think we will need to update the `cdap-build` repo as well after this change is merged.
Since earlier cdap-ui was a submodule of cdap.
cc - @tivv

File: sandboxjs/sandbox_starter.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import zipfile
+
+def run_shell_command(cmd):
+    process = subprocess.run(cmd.split(" "), stderr=subprocess.PIPE)
+    if process.returncode != 0:
+        print("Process completed with error: ", process.stderr)
+    assert process.returncode == 0
+
+# Start CDAP sandbox
+print("Downloading CDAP sandbox")
```
Comment (itsankit-google) [inline sandboxjs/sandbox_starter.py:28]: Optional - Can use python `logging` module instead of `print`.

**Review by sumengwang — APPROVED** [review]

**Pull request merged**